### PR TITLE
Db doctor who

### PR DESCRIPTION
**Pull request by RanaSalameh20**

### CODE REVIEW

File: CreateTheTables.sql [hunk 1]
```diff
@@ -0,0 +1,65 @@
+CREATE TABLE tblEnemy(
```
Comment (hayatzsala) [inline CreateTheTables.sql:3]: There is a small and tricky sentence you missed here at the beginning, Could you guess what it is?

File: changeEpisodeName.sql
Comment (hayatzsala) [inline changeEpisodeName.sql:1]: Could you refactor this query to more simple one?

File: Req8-CreateView.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+CREATE VIEW viewEpisodes AS
+SELECT e.EpisodeId, e.SeriesNumber, e.EpisodeNumber, e.EpisodeType, e.Title, e.EpisodeDate,
```
Comment (hayatzsala) [inline Req8-CreateView.sql]: you can here list all columns by writing e.*, it's better and more dynamically if you add any new column into episode table

**Review by hayatzsala — APPROVED** [review]
Approved with few comments, Good job

File: CreateTheTables.sql (hunk 1 above)
Comment (RanaSalameh20) [reply CreateTheTables.sql:3]: USE DoctorWho;

**Review by hayatzsala — APPROVED** [review]

**Pull request merged**